server/templates/pages/src/400.html

```diff
@@ -1,8 +1,6 @@
 <!doctype html>
-<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
-<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
-<!--[if IE 8]>         <html class="lt-ie9"> <![endif]-->
-<!--[if gt IE 8]><!--> <html class="no-js" dir="{{ lang_dir }}" lang="{{ lang }}"> <!--<![endif]-->
+<!--[if lte IE 8]>      <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
```

Comment:
I promised to get back to you here. I think the right solution for this file for lines R2-R3, and all other error handling files is:

```html
<!--[if lt IE 10]>      <html class="lt-ie10"> <![endif]-->
<!--[if gte IE 10]><!--> <html class="no-js" dir="{{ lang_dir }}" lang="{{ lang }}"> <!--<![endif]-->
```